Automattic/jetpack
# Stats Admin: Add module settings api

Partially Fixes #31228 

## Proposed changes:

The PR proposes to add API to get modules settings on Stats Dashboard, and API to update module settings. The APIs are forwarded to WPCOM.

### Other information:


## Jetpack product discussion
n/a

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:

* Build Odyssey `STATS_PACKAGE_PATH=/path/to/jetpack/projects/packages/stats-admin yarn dev`
* Open `/wp-admin/admin.php?page=stats&flags=stats/highlights-settings`
* Change settings of the Highlight module
* Refresh page
* Ensure settings is persisted

<img width="893" alt="image" src="https://github.com/Automattic/jetpack/assets/1425433/bac62442-fbfe-47ce-9168-396a4ce8e017">
